### PR TITLE
Added PP test underlying regression statistics as properties of the classes

### PR DESCRIPTION
**Pull request by Mill7**

### CODE REVIEW

**Comment by bashtage** [comment]
Is this a bit out of date?  Seems to have functions that are already in the package.

**Comment by Mill7** [comment]
I may be wrong but I can not see how, from an instance of PhilipsPierron, I can get access to the p-values of the regression coefficients for instance ?

**Comment by Mill7** [comment]
> Is this a bit out of date? Seems to have functions that are already in the package.

I may be wrong but I can not see how, from an instance of PhilipsPierron, I can get access to the p-values of the regression coefficients for instance ?

File: arch/unitroot/unitroot.py [hunk 1]
```diff
@@ -1714,3 +1730,63 @@ def kpss_crit(stat, trend='c'):
     crit_value = interp(cv, y[::-1], x[::-1])
 
     return pvalue, crit_value
+
+def BandWidthNW(y, kernel = "ba"):
```
Comment (bashtage) [inline arch/unitroot/unitroot.py:1734]: I was referring to this function.

File: arch/unitroot/unitroot.py [hunk 2]
```diff
@@ -996,6 +1008,10 @@ def _compute_statistic(self):
 
         self._title = self._test_name + ' (Z-' + self._test_type + ')'
 
+        self._coeff_pvalue = resols.pvalues
+
+        self._res_summary = resols.summary()
```
Comment (bashtage) [inline arch/unitroot/unitroot.py:1013]: Shouldn't store the summary.  The result is enough.

File: arch/unitroot/unitroot.py [hunk 3]
```diff
@@ -505,6 +507,16 @@ def trend(self, value):
             self._reset()
             self._trend = value
 
+    @property
```
Comment (bashtage) [inline arch/unitroot/unitroot.py:510]: These "convenience" functions aren't needed IMO. As long as the OLS estimation result is available then it is used to use

adf.regression.pvalue or adf.regression.summary()

**Comment by bashtage** [comment]
You can so the same thign that is done in `ADF`; add to PhillipsPerron

```
@property
def regression(self):
...
```

File: arch/unitroot/unitroot.py (hunk 1 above)
Comment (Mill7) [reply arch/unitroot/unitroot.py:1734]: I did not intend to do anything new on the Bandwidth, sorry.

**Pull request closed without merging**